.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+
+## Fixes 
+<!-- For e.g Fixes: https://issues.redhat.com/browse/HAC-XXX -->
+
+
+## Description
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Type of change
+<!-- Please delete options that are not relevant. -->
```

Comment:
This section probably isn't needed.  The type of fix should be evident from the title, which will refer to a Jira card and/or Bugzilla issue, the changes in the code, and the description.

Reply:
I would like to keep this section because I like seeing the type of change explicitly in the PR description without having to go through the ticket or code changes. This is similar to how we tag our PRs with `/kind bug or /kind feature`.

Reply:
What do you get out of the information provided in this section.
Does it help you do a review or get appropriate reviewers? Why is this useful

Reply:
Yeah, it helps quickly set the context while doing a review. It also helps me prioritize the PRs for review, for example, if I see a big feature PR I would keep it in my backlog to review properly vs if it's a small bug fix I would review it right away and be done with it.

Reply:
Have we decided on using Bugzilla for tracking HAC bugs? I thought we were eventually moving away from Bugzilla to Jira even for the console. 
We also have the `/kind feature`.